pydoctor/templatewriter/util.py

```diff
@@ -17,3 +18,22 @@ def templatefile(filename):
 
 def templatefilepath(filename):
     return FilePath(templatefile(filename))
+
+
+class Page(Element):
```

Comment:
This PR is intended for a bug-fix release. While I appreciate ideas for further cleaning up the code, I think it would be better to consider those for the next major release, and after #299 has landed.